abci/cmd/abci-cli/abci-cli.go

```diff
@@ -477,9 +503,8 @@ func muxOnCommands(cmd *cobra.Command, pArgs []string) error {
 }
 
 func cmdUnimplemented(cmd *cobra.Command, args []string) error {
-	// TODO: Print out all the sub-commands available
 	msg := "unimplemented command"
-	if err := cmd.Help(); err != nil {
+	if err := muxonCmd.Help(); err != nil {
```

Comment:
I think we can do it without any additional commands. What if we just cycle through available commands and print their use and short description

```
fmt.Println("Available commands:")
fmt.Printf("%s: %s\n", echoCmd.Use, echoCmd.Short)
...
```

Reply:
I had made it more complicated.  lol

Thank you!
By the way，I'm reading the code of consensus which is really awesome!!